Intersect.Client/Core/Graphics.cs

```diff
@@ -768,6 +812,76 @@ public static void DrawFullScreenTextureFitMaximum(GameTexture tex)
             }
         }
 
+        private static void DrawFullScreenAnimatedTexture(
+            string animFrameName,
+            int animFrameCount,
+            long animFrameSpeed
+        )
+        {
+            if (animTextureTime < Timing.Global.Milliseconds)
+            {
+                animTextureFrame += 1;
+                animTextureTime = Timing.Global.Milliseconds + animFrameSpeed;
+            }
+
+            if (animTextureFrame >= animFrameCount - 1)
+            {
+                animTextureFrame = 0;
+            }
+
+            var animTex = Globals.ContentManager.GetTexture(
+                Framework.Content.TextureType.Animation, animFrameName + "_" + animTextureFrame + ".png"
+            );
+
+            if (animTex == null)
+            {
+                return;
+            }
+
+            var destRectangle = new FloatRect();
+            var srcRectangle = new FloatRect(0, 0, animTex.Width, animTex.Height);
+            destRectangle.X = 0;
+            destRectangle.Y = 0;
+            destRectangle.Width = srcRectangle.Width;
+            destRectangle.Height = srcRectangle.Height;
+            DrawGameTexture(animTex, srcRectangle, destRectangle, new Color(255, 255, 255, 255));
+        }
+
+        private static void DrawFullScreenAnimatedTextureStretched(
```

Comment:
These two methods are basically the same except for the destination rectangle, the code should get reused.

Reply:
realized that i was writing redundant code, thanks for pointing this out